pyvista/core/pointset.py

```diff
@@ -1707,40 +1708,40 @@ def celltypes(self) -> np.ndarray:
         Returns
         -------
         numpy.ndarray
-            Array of VTK cell types.  Some of the most popular cell types:
-
-        * ``VTK_EMPTY_CELL = 0``
-        * ``VTK_VERTEX = 1``
-        * ``VTK_POLY_VERTEX = 2``
-        * ``VTK_LINE = 3``
-        * ``VTK_POLY_LINE = 4``
-        * ``VTK_TRIANGLE = 5``
-        * ``VTK_TRIANGLE_STRIP = 6``
-        * ``VTK_POLYGON = 7``
-        * ``VTK_PIXEL = 8``
-        * ``VTK_QUAD = 9``
-        * ``VTK_TETRA = 10``
-        * ``VTK_VOXEL = 11``
-        * ``VTK_HEXAHEDRON = 12``
-        * ``VTK_WEDGE = 13``
-        * ``VTK_PYRAMID = 14``
-        * ``VTK_PENTAGONAL_PRISM = 15``
-        * ``VTK_HEXAGONAL_PRISM = 16``
-        * ``VTK_QUADRATIC_EDGE = 21``
-        * ``VTK_QUADRATIC_TRIANGLE = 22``
-        * ``VTK_QUADRATIC_QUAD = 23``
-        * ``VTK_QUADRATIC_POLYGON = 36``
-        * ``VTK_QUADRATIC_TETRA = 24``
-        * ``VTK_QUADRATIC_HEXAHEDRON = 25``
-        * ``VTK_QUADRATIC_WEDGE = 26``
-        * ``VTK_QUADRATIC_PYRAMID = 27``
-        * ``VTK_BIQUADRATIC_QUAD = 28``
-        * ``VTK_TRIQUADRATIC_HEXAHEDRON = 29``
-        * ``VTK_QUADRATIC_LINEAR_QUAD = 30``
-        * ``VTK_QUADRATIC_LINEAR_WEDGE = 31``
-        * ``VTK_BIQUADRATIC_QUADRATIC_WEDGE = 32``
-        * ``VTK_BIQUADRATIC_QUADRATIC_HEXAHEDRON = 33``
-        * ``VTK_BIQUADRATIC_TRIANGLE = 34``
+            Array of cell types.  Some of the most popular cell types:
```

Comment:
I mean in this function code.  I could only comment on the changed lines in the docstring.

Reply:
Thanks for the suggestion. Do you mean the string of CellType?

Reply:
I'm not sure returning an array of enum values is a good idea, because that would only work with an object-dtype array which we should avoid. We could return a list of enum values, but I'm not sure how practical that is downstream.